bucket/selenium.json

```
@@ -1,16 +1,22 @@
 {
     "version": "3.141.59",
-    "homepage": "https://www.seleniumhq.org/",
-    "description": "A portable framework for testing web applications.",
+    "homepage": "https://selenium.dev",
+    "description": "A smart proxy server that allows Selenium tests to route commands to remote web browser instances",
```

```suggestion
    "description": "A browser automation framework and ecosystem",
```

See https://github.com/SeleniumHQ/selenium

---

This manifest is only for Selenium Server (Grid).  The description in github repo is for all the Selenium components.